Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1520,7 +1520,7 @@ static void webProcessPoolHighDynamicRangeDidChangeCallback(CFNotificationCenter
 
     RELEASE_LOG(Process, "WebProcessPool::registerUserInstalledFonts: start registering fonts");
     RetainPtr requestedProperties = [NSSet setWithArray:@[@"NSFontNameAttribute", @"NSFontFamilyAttribute", @"NSCTFontFileURLAttribute", @"NSCTFontUserInstalledAttribute"]];
-    RetainPtr fontProperties = adoptCF(XTCopyPropertiesForAllFontsWithOptions(bridge_cast(requestedProperties.get()), kXTScopeAll, kXTOptionsDoNotSortResults));
+    RetainPtr fontProperties = adoptCF(XTCopyPropertiesForAllFontsWithOptions(bridge_cast(requestedProperties.get()), kXTScopeGlobal, kXTOptionsDoNotSortResults));
```

Comment:
What does this flag mean? We may want to include it in commit message.

Reply:
This flag means we only get the fonts available globally from the fonts service, not local fonts, like fonts in apps.
I will include it in the commit message.

Thanks for reviewing!